qemu/tests/block_copy.py

```diff
@@ -456,3 +450,50 @@ def reopen(self, reopen_image):
         self.params["image_name_%s" % self.tag] = reopen_image
         self.vm.create(params=self.params)
         self.vm.verify_alive()
+
+    def hot_unplug(self):
```

Comment:
The mean of this name is not clear.

After reading the following approach, it looks like be a separate "test", so I think here is not the right place to put it in.